test/integration/crcsuite/prepare.go

```diff
@@ -67,9 +68,14 @@ func CopyFilesToTestDir() {
 	}
 
 	l := strings.Split(cwd, string(filepath.Separator))
-	dataDirPieces := append([]string{string(filepath.Separator)}, l[:len(l)-3]...)
-	dataDirPieces = append(dataDirPieces, "testdata")
+	dataDirPieces := append(l[:len(l)-3], "testdata")
+	var volume string
+	if runtime.GOOS == "windows" {
+		volume = filepath.VolumeName(cwd)
+		dataDirPieces = dataDirPieces[1:] // drop volume from list of dirs
+	}
 	dataDir := filepath.Join(dataDirPieces...)
+	dataDir = fmt.Sprintf("%s%c%s", volume, filepath.Separator, dataDir) // prepend volume back
```

Comment:
`dataDir := filepath.Join(volume, filepath.Separator, dataDir)`? or is this not working as expected when volume is empty?

Reply:
It should work fine with any volume (didn't test), but it's a hack: we pretend that `\` is a folder name, then `filepath.Join` creates a path `C:\\\datadir` and simplifies it to `C:\datadir`. We'd also have to cast `filepath.Separator` to `string`. Going to test it properly now...